packages/amplify_core/lib/src/types/api/graphql/graphql_response.dart

```diff
@@ -22,12 +22,12 @@ class GraphQLResponse<T> {
   /// This will be `null` if there are any GraphQL errors during execution.
   final T? data;
 
-  /// A list of errors from execution. If no errors, it will be an empty list.
-  final List<GraphQLResponseError> errors;
+  /// A list of errors from execution. If no errors, it will be `null`.
+  final List<GraphQLResponseError>? errors;
```

Comment:
What was the reason for making this nullable again? Nullable lists are a pain to work with especially when an empty list and null list mean the same thing.

Reply:
graphql spec "The errors entry in the response is a non‐empty list of errors, where each error is a map.

If no errors were encountered during the requested operation, the errors entry should not be present in the result." https://spec.graphql.org/June2018/#sec-Errors says

Reply:
just pushed this change